### DISCUSSION TITLE
Integrazione Ezviz: stream network error e latenza

### DISCUSSION THREAD

**Discussion by mikelegnr** [indomus/forum#2417] [HUB: Home Assistant]
Buongiorno a tutti, ho integrato le videocamere Ezviz C3WN su home assistant, attraverso l'integrazione ufficiale. Le immagini si vedono ma riscontro due problemi:

- dopo qualche minuto il video si blocca e appare la scritta "Stream network error". Aggiornando la dashboard riparte tutto, ma ad un certo punto il problema si ripresenta;

- le immagini presentano un ritardo di circa 6 secondi.

Qualcuno ha risolto? 
Vi ringrazio in anticipo

**Comment by indomus** [comment]
> Buongiorno a tutti, ho integrato le videocamere Ezviz C3WN su home assistant, attraverso l'integrazione ufficiale. Le immagini si vedono ma riscontro due problemi:
> 
> * dopo qualche minuto il video si blocca e appare la scritta "Stream network error". Aggiornando la dashboard riparte tutto, ma ad un certo punto il problema si ripresenta;
> * le immagini presentano un ritardo di circa 6 secondi.
> 
> Qualcuno ha risolto? Vi ringrazio in anticipo

https://github.com/home-assistant/core/issues/new/choose

**Reply by Pigi1966** [reply]
Beato te, sei ad un gran punto. Io ieri ho cercato di integrare su HA due EZVIZ BC2 purtroppo senza esito mi da un errore in fase di configurazione “ Nome Host o indirizzo IP non valido “

**Comment by SkunkGuru** [comment]
> * dopo qualche minuto il video si blocca e appare la scritta "Stream network error". Aggiornando la dashboard riparte tutto, ma ad un certo punto il problema si ripresenta;

io ho questo problema se guardo il video dal browser del pc, ma dalla app di HA funziona correttamente

prova a integrarle direttamente come flusso onvif magari
